src/statistics.jl

```diff
@@ -1,28 +1,46 @@
-# Autocorrelation for a specifig lag
-function autocor(x::AbstractVector, lag::Integer)
+"""
+    autocor(x,lag)
```

Comment:
Any specific reason I miss for calling it `lag`?

Reply:
It is the same name that is being used in StatsBase.jl - we could call it something else. Calling it ```time``` or ```t``` could be misleading but maybe something like ```steps``` or ```timesteps```?

Reply:
Okay yes makes sense to stick to something that has already been around for some time.